ksqldb-execution/src/test/java/io/confluent/ksql/execution/util/ExpressionTypeManagerTest.java

```diff
@@ -205,13 +206,16 @@ public void shouldFailIfComparisonOperandsAreIncompatible() {
     );
 
     // When:
-    final Exception e = assertThrows(
-        KsqlException.class,
+    final KsqlStatementException e = assertThrows(
+        KsqlStatementException.class,
         () -> expressionTypeManager.getExpressionSqlType(expr)
     );
 
     // Then:
     assertThat(e.getMessage(), containsString(
+        "Cannot compare BIGINT to STRING with GREATER_THAN"
+    ));
+    assertThat(e.getUnloggedMessage(), containsString(
```

Comment:
In general, you should expect to see that the `message` has changed to drop all query text, while the new `unloggedMessage` is the same as the previous message (which means that the API reponses will be the same as in the past).